packages/contracts-bedrock/scripts/L2Genesis.s.sol

```diff
@@ -154,60 +161,44 @@ contract L2Genesis is Script, Artifacts {
     /// @dev Sets all the implementations for the predeploy proxies. For contracts without proxies,
     ///      sets the deployed bytecode at their expected predeploy address.
     function _setPredeployImplementations() internal {
-        _setLegacyMessagePasser();
-        _setDeployerWhitelist();
-        _setWETH9();
-        _setL2StandardBridge();
+        _setL2ToL1MessagePasser();
```

Comment:
It seems we changed the order of some of the existing ones were changed, what's the reason for that?

Reply:
Yea sorry it looks messy in the file changes. Was ordering them as they're specified in [Predeploys.sol](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/src/libraries/Predeploys.sol) to make it easier to check we have them all